src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -835,6 +845,15 @@ public function run() {
     return 0;
   }
 
+  // Default value for $recursive = true so we do not break existing behavior
+  private function submoduleUpdate($repository_api, $recursive = true) {
+    $command = 'submodule update --init';
+    if($recursive)
+      $command .= ' --recursive';
```

Comment:
Good point. I don't really like it, but if it's the language convention it probably makes sense to leave it in.